LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -137,7 +137,12 @@ function addHeader(column) {
                     $('#'+data.cellContainerID).html(blob[data.recordID].recordResolutionData.lastStatus);
                 }
             }});
-        break;
+            headers.push({name: 'Resolved By', indicatorID: 'resolvedBy', editable: false, callback: function(data, blob) {
```

Comment:
This isn't pulling anything as the indicatorID: resolvedBy does not exist, also there is no front-end for the user to select this as a header on the report builder that I can see, unless I'm missing it.

![image](https://user-images.githubusercontent.com/17956374/121242570-b729f180-c86a-11eb-8b25-e1480ab00876.png)

Reply:
Correct because it is part of the resolved by info, ie: date/approved/name. I had tested it alone, but it is part of the resolved by section

Reply:
this is not done